cmd/commands/server/command_server.go

```diff
@@ -83,8 +93,18 @@ func (cmd *Command) Start() (err error) {
 	}
 	go func() {
 		for {
-			time.Sleep(1 * time.Minute)
-			cmd.mysteriumClient.PingProposal(proposal, signer)
+			select {
+			case <-time.After(1 * time.Minute):
```

Comment:
I would externalise various timeouts as config parameters, like: 

const commandServerPingTimeout := 1 * time.Minute

Reply:
Yes, but this is not timeout. It's ping period. And it's used in one place only